source/build.gradle.kts

```diff
@@ -40,15 +43,66 @@ publishing {
             artifactId = "source-android"
             version = libs.versions.libraryVersion.get()
 
+            pom {
+                name.set("Source Android")
+                description.set("Guardian design system library for Android")
+                url.set("https://github.com/guardian/source-android")
+                packaging = "aar"
+                licenses {
+                    license {
+                        name.set("Apache License, Version 2.0")
+                        url.set("https://github.com/guardian/source-android/tree/main?tab=Apache-2.0-1-ov-file#readme")
+                    }
+                }
+                developers {
+                    developer {
+                        id.set("ab-gnm")
+                        name.set("Aditya Bhaskar")
+                        email.set("aditya.bhaskar@guardian.co.uk")
+                        url.set("https://github.com/ab-gnm")
+                    }
+                }
+                organization {
+                    name.set("Guardian News & Media")
+                    url.set("https://www.theguardian.com")
+                }
+                scm {
+                    connection.set("scm:git:github.com/guardian/source-android.git")
+                    developerConnection.set("scm:git:ssh://github.com/guardian/source-android.git")
+                    url.set("https://github.com/guardian/source-android/tree/main")
+                }
+            }
+
+            // Use the artifacts called "release" for publishing.
             afterEvaluate {
                 from(components["release"])
             }
         }
     }
+
     repositories {
+        maven {
+            name = "sonatype"
+            url = uri("https://s01.oss.sonatype.org/service/local/staging/deploy/maven2/")
```

Comment:
I think you may want `oss.sonatype.org` rather than `s01.oss.sonatype.org` here. The business of alternative hosts, and who is using what, is confusing, but I tried to summarise it [here](https://github.com/xerial/sbt-sonatype/pull/461):

> ### Sonatype host is determined by-profile (_not_ by-user)
> 
> Note that in the [past](https://github.com/xerial/sbt-sonatype/issues/273), there's been a bit of confusion (ok, _I've_ been a bit confused!) about whether hosts are assigned on a by-user or by-profile basis (many users can be granted access by Sonatype to upload to one profile, eg we at the Guardian have that for our com.gu profile). It's definitely [per-profile](https://github.com/xerial/sbt-sonatype/issues/273#issuecomment-997486538) though - so `sonatypeProfileName` dictates which `sonatypeCredentialHost` you want to be using.
> 
> As a user (eg, for me, with my Sonatype user account 'roberto') working on a variety of projects with different profiles (eg, for me, working on projects under `com.gu`, `com.madgag` & `org.scanamo`) when I'm performing a release I'll be uploading to either `oss.sonatype.org`, _or_ `s01.oss.sonatype.org`, depending on which profile/groupId that particular project lives under.


You should be uploading to the `com.gu` Sonatype profile, and your artifacts should have a group coordinate which is either `com.gu`, or some subspace of that, ie `com.gu.source` as you've got configured:

https://github.com/guardian/source-android/blob/f16239c159f76cd44c2460618a637277d91ae693/source/build.gradle.kts#L42

As the `com.gu` Sonatype profile currently belongs to the `oss.sonatype.org` host, that's where you'll want your artifacts to go - not `s01.oss.sonatype.org`!